pkg/asset/agent/image/baseiso.go

```diff
@@ -129,6 +130,10 @@ func (i *BaseIso) Generate(dependencies asset.Parents) error {
 			return nil
 		}
 		if !errors.Is(err, &exec.Error{}) { // Already warned about missing oc binary
+			if strings.Contains(err.Error(), "no such file or directory") {
```

Comment:
This looks pretty risky, we don't even control that error message.

Wouldn't `os.IsNotExist(err)` (or I guess more properly `errors.Is(err, fs.ErrNotExist)`) actually do what we need?

Reply:
Yep, that's better. Changed.